api.bs

```diff
@@ -613,15 +617,95 @@ the last (most recent) impression that matches the [[#logic-matching|common matc
 The entire [=conversion value=] (up to the maximum imposed by the [[#dp-budget|privacy budget]])
 is allocated to the histogram bucket that was saved with the impression.
 
+Last touch attribution does not select any impression
+that was saved during a week
+that does not have sufficient [=privacy budget=].
+If impressions match from a week
+that does not have enough [=privacy budget=],
+impressions are not matched for any preceding weeks.
+That is, once a week has a matching impression
+and insufficient budget,
+the process will set a value of zero for all histogram buckets.
+
+To <dfn ignore>fill a histogram using last-touch attribution</dfn>,
+given |options|:
+
+1.  Initialize |impression| to a null value.
+
+1.  Initialize |value| to |options|.|value|.
+
+1.  Let |now| be the current time.<!-- TODO: cite HRTIME spec -->
+
+1.  For each |week| starting from the current week
+    to the oldest week supported by the [=user agent=]:
```

Comment:
We should define somewhere what a week is. Presumably UTC, but need to choose Sunday or Monday.